hypha/settings/base.py

```diff
@@ -478,3 +478,5 @@
         debug=SENTRY_DEBUG,
         integrations=[DjangoIntegration()]
     )
+
+DEFAULT_REVIEW_VISIBILITY = env.str('DEFAULT_REVIEW_VISIBILITY', 'private')
```

Comment:
Please rename to `REVIEW_VISIBILITY_DEFAULT` and move it up to the `# Hypha custom settings` section.